python/vTestInterface.py

```diff
@@ -391,7 +392,9 @@ def executeVCtest(enviroPath, testIDObject, generateReport):
         returnText = ""
 
         returnCode, commandOutput = clicastInterface.executeTest(testIDObject)
-        if generateReport:
+        # return code 98 means the the coded test failed to compile so
```

Comment:
Should this comment not be against the variable declaration rather than its usage?